pycqed/measurement/measurement_control.py

```diff
@@ -1270,7 +1519,7 @@ def print_progress(self, stop_idx=None):
             percdone = self.get_percdone()
             elapsed_time = time.time() - self.begintime
             progress_message = "\r {percdone}% completed \telapsed time: "\
-                "{t_elapsed}s \ttime left: {t_left}s".format(
+                "{t_elapsed}s \ttime left: {t_left}s     ".format(
```

Comment:
Why don't you use \t\t at the end of this string instead of many spaces?

Reply:
I am not sure whether \t\t will correctly overwrite old output on every kind of terminal. The many spaces should work for sure.

Reply:
But you already use \t in that string. So by that argument shouldn't you remove all \t's?

Reply:
"you" is not really true, it has been there before. My understanding is: The \t introduces visible white space that corresponds to multiple spaces. This is fine for the first use of \t. But in a string, \t counts only as a single character (ASCII code 9). So I suspect that it will only overwrite a single character and not multiple characters. The result might be a big visible white space and some leftover character(s) afterwards. I do not see any disadvantage of using multiple space (ASCII code 32) characters to be sure that no old characters are left over. If you think it is worth investing time to test whether \t does the same job, I will leave the PR open until you have tested it.

Reply:
I was mostly just curious about the difference between using one \t and multiple.